docs/src/content/extra/detailed.md

```diff
@@ -0,0 +1,206 @@
+
+### Bots Done Better
+
+Any user can make a bot user which is associated with their account.
+
+Bot users are more or less treated like normal users with a few differences.
+
+1. Bots can register Application commands and message components.
+2. Bots cannot have friends. (I tried to convince the other Eludritians but they refused. - Enoki :( )
+3. Bots cannot join communities themselves, however, they can be invited into them unlike normal accounts.
+4. Bots will not have multiple session tokens and instead will have one token that they can regenerate.
+
+Verification for bots only means that the bot and its owner will be given a little neat badge of honour.
+
+Discord-styled application commands will be available. However -- unlike Discord -- they will not be forced upon people and will have more cool features, uses and will be more flexible. Additionally buttons and more message components will be available to give bot developers more room and tools to make cool stuff.
+
+## Miscellaneous Info
+
+### IDs
+
+A Eludris ID is a 128 bit (16 byte) number, structured like so:
+
+```
+ 12345678  12345678  12345678  12345678  12345678  12345678  12345678  12345678  12345678  12345678  12345678  12345678  12345678  12345678  12345678  12345678
+ TTTTTTTT  TTTTTTTT  TTTTTTTT  TTTTTTTT  TTTTTTTT  TTTTTTTT  TTTTTTTT  TTTTTTTT  IIIIIIII  IIIIIIII  IIIIIIII  IIIIIIII  IIIIIIII  IIIIIIII  SSSSSSSS  SSSSSSSS
+╰──────────────────────────────────────────────────────────────────────────────╯╰──────────────────────────────────────────────────────────╯╰──────────────────╯
+                                       │                                                                     │                                       │
+                                       │                                                                     │                          16 bit (2 byte) sequence.
+                                       │                                                       48 bits (6 byte) Instance ID.
+                        64 bit (8 byte) Unix timestamp.
+```
+
+T: A Unix timestamp with the Eludris epoch (1,650,000,000)─.
+
+I: The id of the instance that generated this ID.
+
+S: The sequence number of this ID
+
+An instance ID is a 48 bit (6 byte) number, structured like so:
+
+```
+ 12345678  12345678  12345678  12345678  12345678  12345678  12345678 12345678
+ TTTTTTTT  TTTTTTTT  TTTTTTTT  TTTTTTTT  TTTTTTTT  TTTTTTTT  TTTTTTTT TTTTTTTT
+╰─────────────────────────────────────────────────────────────────────────────╯
+                                       │
+                                       │
+                        48 bit (6 byte) Unix timestamp.
+```
+
+T: 48 bits of the current Unix timestamp (also with the Eludris) epoch.
+
+### KeyDB
+
+Eludris uses a non persistent KeyDB instance to store data that should be fetched with low latency, and is ephemeral such as rate-limit info.
```

Comment:
I'd change this to "<...> to store data that should be fetched with low latency and is ephemeral, such as rate-limit info."
(Note, I moved the comma)